src/components/CreateInstructorPage/index.jsx

```diff
@@ -0,0 +1,169 @@
+  setStartedFetching() {
+    this.setState({ startedFetching: true });
+  }
+
+  processSocialLinks(links) {
+    if (!links) {
+      return [];
+    }
+
+    return links.map(link => (
+      ({
+        id: null,
+        type: link.type,
+        title: link.title,
+        url: link.url,
+      })
+    ));
+  }
+
+  processAreasOfExpertise(areasOfExpertise) {
+    if (!areasOfExpertise) {
+      return [];
+    }
+
+    return areasOfExpertise.map(area => ({ id: null, value: area.value }));
+  }
+
+  processPosition(title, organization) {
+    return {
+      title,
+      organization,
+      organization_override: null,
+    };
+  }
+
+  handleInstructorCreate(options) {
+    const instructorData = {
+      profile_image: options.image,
+      given_name: options.firstName,
+      family_name: options.lastName,
+      bio: options.bio,
+      major_works: options.majorWorks,
+      areas_of_expertise: this.processAreasOfExpertise(options.areasOfExpertise),
+      urls_detailed: this.processSocialLinks(options.socialLinks),
+      position: this.processPosition(options.title, options.organization),
+    };
+
+    this.props.createInstructor(instructorData);
+  }
+
+  render() {
+    const {
+      instructorOptions,
+      instructorInfo,
+    } = this.props;
+
+    if (!instructorOptions || !instructorInfo) {
+      return (
+        <StatusAlert
+          id="error"
+          alertType="danger"
+          title="Create Instructor Form failed to load: "
+          message="Instructor options information unavailable."
+        />
+      );
+    }
+
+    const { startedFetching } = this.state;
+    const { isFetching } = instructorOptions;
+    const showSpinner = !startedFetching || isFetching;
+    const showForm = startedFetching && !isFetching;
+
+    const { isCreating } = instructorInfo;
+
+    let error = '';
+    if (instructorInfo.error) {
+      error = error.concat(instructorInfo.error, ' ');
+    }
+    if (instructorOptions.error) {
+      error = error.concat(instructorOptions.error);
+    }
+    error = error.trim();
+
+    return (
+      <React.Fragment>
+        <Helmet>
+          <title>Create a New Instructor</title>
+        </Helmet>
+        <div className="container-fluid">
+          <div className="row justify-content-md-center my-3 ">
+            <div className="col-6">
+              { showSpinner && <LoadingSpinner /> }
+              { showForm && (
+                <div>
+                  <h2>Create New Instructor</h2>
+                  <hr />
+                  <h3>Instructor Information</h3>
+                  <div className="col">
+                    <CreateInstructorForm
+                      id="create-instructor-form"
+                      onSubmit={this.handleInstructorCreate}
+                      instructorOptions={instructorOptions}
+                      isCreating={isCreating}
+                    />
+                    { isCreating && <LoadingSpinner message="Creating Instructor" /> }
```

Comment:
Feels like the two LoadingSpinners could be merged into one, but I lost interest in this feedback halfway through writing it. The code is perfect, don't change a thing.